lib/Parser/OglExtension.php

```diff
@@ -120,10 +121,56 @@ public function incorporate($other, $commit) {
         }
     }
 
+    /**
+     * Add a sub-extension, or merge it if it already exists.
+     *
+     * @param string $name Name of the extension.
+     * @param string $status Status of the extension.
+     * @param array $supportedDrivers List of drivers supported for this extension.
+     * @param #Mesamatrix\Git\Commit $commit The commit used by the parser.
```

Comment:
Should I remove `\Mesamatrix\` in here too and keep only `Git\Commit` since it's the same base namespace?

(the same goes for the other lines where I do this kind of reference)